infl-components-src/tooltip/index.js

```diff
@@ -59,7 +59,7 @@ export default class Tooltip extends Component {
   static propTypes = {
     title:             PropTypes.string,
     element:           PropTypes.node.isRequired,
-    position:          PropTypes.oneOf(['top', 'bottom']),
+    position:          PropTypes.string,
```

Comment:
Yes but since the internal component checks it anyways, do we need to duplicate it?